src/rules/arengu-progressive-profiling.js

```diff
@@ -0,0 +1,76 @@
+/**
+ * @title Arengu Progressive Profiling 
+ * @overview Capture new users' information in your authentication flows.
+ * @gallery true
+ * @category marketplace
+ */
+
+ async function arenguCompleteUserProfile(user, context, callback) {
+  if (
+    !configuration.SESSION_TOKEN_SECRET ||
+    !configuration.ARENGU_POLICIES_FORM_URL
+  ) {
+    console.log('Missing required configuration. Skipping.');
+    return callback(null, user, context);
+  }
+
+  const {
+    Auth0RedirectRuleUtilities,
+    Auth0UserUpdateUtilities
+  } = require("@auth0/rule-utilities@0.2.0");
+
+  const ruleUtils = new Auth0RedirectRuleUtilities(
+    user,
+    context,
+    configuration
+  );
+
+  const userUtils = new Auth0UserUpdateUtilities(user, auth0);
+
+  function validateSessionToken() {
+    try {
+      return ruleUtils.validateSessionToken();
+    } catch (error) {
+      callback(error);
+    }
+  }
+
+  // Modify your login criteria to your needs
+  function isLogin() {
+    return context.stats.loginsCount > 2;
```

Comment:
I would suggest the `2` here being a Rule configuration, like:

```
function isLogin() {
  const loginCountForProfiling = configuration.ARENGU_PROFILE_LOGIN_COUNT || 2;
  return context.stats.loginsCount > parseInt(loginCountForProfiling, 10);
}
```

Reply:
Good one! Done!